python/cm/models.py

```diff
@@ -876,6 +876,12 @@ def get_id_chain(self, target_ids: dict) -> dict:
 
     def allowed(self, obj: ADCMEntity) -> bool:
         """Check if action is allowed to be run on object"""
+        if self.state_unavailable == 'any':
+            return False
+
+        if self.multi_state_unavailable == 'any':
+            return False
+
```

Comment:
So we drop support of ANy here?
@acmnu Are you ok with it?

Reply:
Otherwise, I added support for 'any' in unavailable, as it was discussed.

Reply:
Ok. Thx for clarifying